### PR TITLE
import empty calendars

### LINKED ISSUE #194: Importing empty Calendars
### Steps to reproduce
1. Create an empty calendar
2. export that Calendar
3. import that Calendar to new instance of Nextcloud

### Expected behaviour
It should just complete the import bring nothing in

### Actual behaviour
It hangs saying that it is scheduling the import and just stays there.

### Server configuration
Ubuntu 16.04:

**Web server:** Apache2

**Database:** Mysql

**PHP version:** 7

**Server version:** (see your admin page) 10.1

**Calendar version:** (see the apps page) Latest

**Updated from an older installed version or fresh install:** No

**Signing status (ownCloud/Nextcloud 9.0 and above):**

```
Login as admin user into your cloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your instance's installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your instance's installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your instance's installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**

**Operating system:**

**CalDAV-clients:**

#### Web server error log
```
Insert your webserver log here
```

#### Log file (data/owncloud.log)
```
Insert your ownCloud.log file here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

### PR DESCRIPTION
**Pull request by georgehrke**
fixes #194 

please review @tcitworld @nextcloud/calendar

### CODE REVIEW

**Comment by tcitworld** [comment]
Perfect. 👍

**Comment by skjnldsv** [comment]
Seems good but build failed?

**Comment by georgehrke** [comment]
that was a travis error at that time. I started the tests again and now they pass.

File: js/app/controllers/importcontroller.js [hunk 1]
```diff
@@ -125,10 +125,10 @@ app.controller('ImportController', ['$scope', '$filter', 'CalendarService', 'VEv
...
 
-				var calendar = $scope.calendars.filter(function (element) {
+				const calendar = $scope.calendars.find(function (element) {
 					return element.url === fileWrapper.selectedCalendar;
-				})[0];
-				if (calendar.enabled) {
+				});
+				if (calendar && calendar.enabled) {
 					calendar.enabled = false;
 					calendar.enabled = true;
```
Comment (skjnldsv) [inline js/app/controllers/importcontroller.js]: What is the point of this?

File: js/app/controllers/importcontroller.js [hunk 2]
```diff
@@ -125,10 +125,10 @@ app.controller('ImportController', ['$scope', '$filter', 'CalendarService', 'VEv
...
 				$scope.closeIfNecessary();
 
-				var calendar = $scope.calendars.filter(function (element) {
+				const calendar = $scope.calendars.find(function (element) {
 					return element.url === fileWrapper.selectedCalendar;
-				})[0];
-				if (calendar.enabled) {
+				});
+				if (calendar && calendar.enabled) {
 					calendar.enabled = false;
```
Comment (skjnldsv) [inline js/app/controllers/importcontroller.js]: What is the point of this?

File: js/app/controllers/importcontroller.js (hunk 1 above)
Comment (skjnldsv) [inline js/app/controllers/importcontroller.js]: What is the point of this?
only one is needed right?

Comment (georgehrke) [inline js/app/controllers/importcontroller.js]: This triggers a rerendering of the events in this calendar

Comment (skjnldsv) [inline js/app/controllers/importcontroller.js]: That what I thought! :)

**Comment by skjnldsv** [comment]
👍

**Pull request merged**